flight/mcu/platformio.ini

```diff
@@ -74,8 +73,7 @@ board = teensy30
 framework = arduino
 upload_protocol = teensy-cli
 
-lib_deps =
-	PaulStoffregen/PWMServo
+lib_deps = PaulStoffregen/PWMServo
```

Comment:
Might look into adding all libraries through here if possible for the next launch instead of manually putting them in the lib folder. **SEE**: be careful while doing this because it's possible we've made minor changes to some of the libraries. I remember changing something about an SPI object within the Radiohead library we use for telemetry.